### PR TITLE
Add 64-bit versions of JsonObjectAppendInteger and JsonCreateInteger

### PR DESCRIPTION
**Pull request by oleorhagen**

### CODE REVIEW

File: tests/unit/json_test.c [hunk 1]
```diff
@@ -922,6 +922,7 @@ static void test_parse_escaped_string(void)
 static void test_parse_big_numbers(void)
 {
 #define JSON_TEST_BIG_NUMBER "9999999999"
+#define JSON_TEST_BIG_NUMBER_INT64 9999999999
```
Comment (vpodzime) [inline tests/unit/json_test.c:925]: Should be `9999999999L`, I think.

**Review by vpodzime — COMMENTED** [review]
Looks good to me otherwise.

**Review by kacf — DISMISSED** [review]

**Comment by olehermanse** [comment]
@cf-bottom jenkins

**Review by olehermanse — DISMISSED** [review]

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/6165/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/6165/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-6165/

File: tests/unit/json_test.c (hunk 1 above)
Comment (oleorhagen) [reply tests/unit/json_test.c]: Fixed with the `LL`  suffix, as discussed in private.

**Review by vpodzime — APPROVED** [review]
Looks good to me.

**Review by olehermanse — APPROVED** [review]

**Pull request merged**